app/views/works/_curator_controlled.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="section-title">
```

Comment:
I wonder why this is a complete add. https://github.com/pulibrary/pdc_describe/blob/main/app/views/works/_curator_controlled_form.html.erb exists in main.  If you want to rename it, that would be ok.